File: src/PrivateVehicle.java
public class PrivateVehicle extends Vehicle {
    private int seatCount;
    public PrivateVehicle(int year, String brand, String model, String spz, int seatCount) {
        super(year,brand,model,spz);
        this.seatCount = seatCount;
    }
    public int getSeatCount() {
        return seatCount;
    }
    public void setSeatCount(int seatCount) {
        this.seatCount = seatCount;
    }

    @Override
    public void displayInfo(){
        super.displayInfo();
        System.out.println("Private vehicle with: " + seatCount + " seats");
    }

}

File: src/PrivateVehicle.kt
class PrivateVehicle(year: Int, brand: String, model: String, spz: String, var seatCount: Int) :
    Vehicle(year, brand, model, spz) {
    public override fun displayInfo() {
        super.displayInfo()
        println("Private vehicle with: $seatCount seats")
    }

    override fun drive() {
        println("Private vehicle $brand $model $spz have $seatCount seats")
    }
}